requirements.txt

```diff
@@ -3,6 +3,7 @@ aws-requests-auth==0.4.3
 boto3==1.40.3  # Match version of the `boto3-stubs` dev dependency
 botocore==1.40.3
 chevron==0.14.0  # Match with types-chevron in requirements.dev.txt
+cryptography==45.0.6 # Pinning to avoid 'Ambiguous transitive … versions'
```

Comment:
I'd rather not pin this security-sensitive dependency. Which direct dependencies cause the ambiguity?